Apps/Start-Products/Start11 v2/es.lng

```diff
@@ -881,18 +881,24 @@ Exist=1
 
 ; Added in 2.05
 "Secondary display when > 1 screen"="Pantalla secundaria cuando hay > 1 pantalla"
-"Pin website..."="Anclar sitio web"
+"Pin website..."="Anclar sitio web..."
```

Comment:
One of the reasons I tend to leave off trailing '...' in translations is that fitting verbose languages is already challenging.  If it fits here, great.